test/e2e/framework/kcp.go

```diff
@@ -385,7 +385,7 @@ func (c *kcpServer) defaultConfig() (*rest.Config, error) {
 		return nil, err
 	}
 
-	config := clientcmd.NewNonInteractiveClientConfig(raw, "system:admin", nil, nil)
+	config := clientcmd.NewNonInteractiveClientConfig(raw, TestConfig.AdminContext(), nil, nil)
```

Comment:
now I am confused. The context we want here is the base context, no? So we need `TestConfig.BaseConfig()` See my other comments about RootShard...Context. Do we need both?

Reply:
see https://github.com/kcp-dev/kcp/pull/1448#discussion_r913587266

Reply:
resolved, this PR now goes into the right direction